server/api.js

```diff
@@ -79,29 +77,31 @@ router.post("/energizers", async (req, res) => {
 	if (!name || !description || !rating) {
```

Comment:
I know this is not being added by this PR, but why does the post method accept a rating at all?

Reply:
as our energiser card on energiser page has rate displaying on then , when we posted new energisers it would display rate:NaN , on the card so we discussed it on Saturday while Patrik was with us an we decide to add rate on energiser post form to prevent this issue.

Reply:
Okay, that's strange - if that's an issue the correct place to solve it is in the presentation layer (frontend) or maybe in the GET endpoint where you can just return a default average of 5. Not by inserting irrelevant data(since that rating comes from the creator) into the database.